### PR TITLE
Gitissue4686

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

**Comment by davidmu1** [comment]
#sign-off

File: api-reference/v1.0/api/user-list.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ This method supports the [OData Query Parameters](https://developer.microsoft.co
...
+To return an alternative property set, specify the desired set of [user](../resources/user.md) properties using the OData `$select` query parameter. For example, to return _displayName_, _givenName_, and _postalCode_, add the following to your query `$select=displayName,givenName,postalCode`.
 
-> Note: Certain properties cannot be returned within a user collection. The following properties are only supported when [retrieving an single user](./user-get.md): _aboutMe, birthday, hireDate, interests, mySite, pastProjects, preferredName, responsibilities, schools, skills, mailboxSettings_
+> Note: Certain properties cannot be returned within a user collection. The following properties are only supported when [retrieving an single user](./user-get.md): _aboutMe, birthday, hireDate, interests, mySite, pastProjects, preferredName, responsibilities, schools, skills, mailboxSettings_.
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list.md]: Should we use the [!NOTE] syntax here?

**Comment by davidmu1** [comment]
Removed the note callout.

#sign-off

File: api-reference/v1.0/api/user-list.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ GET /users
 
 This method supports the [OData Query Parameters](https://developer.microsoft.com/graph/docs/concepts/query_parameters) to help customize the response.
 
-By default, only a limited set of properties are returned ( _businessPhones, displayName, givenName, id, jobTitle, mail, mobilePhone, officeLocation, preferredLanguage, surname, userPrincipalName_ ). 
+By default, only a limited set of properties are returned (**businessPhones**, **displayName**, **givenName**, **id**, **jobTitle**, **mail**, **mobilePhone**, **officeLocation**, **preferredLanguage**, **surname**, and **userPrincipalName**).
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list.md]: FYI, I applied bold to the property names, per the Microsoft Writing Style Guide. Just something to be on the lookout for.

**Pull request merged**